test/data/state-golden-files/project-ds/project-ds-limits.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Doesn't apply here, but is it possible for secrets to leak as a result here? For instance, how will the golden file handle exporting terraform state for sensitive fields?

Reply:
This a good point. Within the context of acceptance tests, I see sensitive values are handled 2 ways:

- Dummy values are defined (example: [data_source_mongodbatlas_database_user_test.go#L52](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/data_source_mongodbatlas_database_user_test.go#L52)) which would not be an issue
- Make use of env variables (example: [resource_mongodbatlas_alert_configuration_test.go#L333](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_alert_configuration_test.go#L333)). This case is more concerning because the person creating the test could serialize in the json the value used for testing locally. Will look into this to see if they can be filtered automatically.